example/19_gemm_softmax/gemm_softmax_xdl_fp16.cpp

```diff
@@ -0,0 +1,551 @@
+#include <iostream>
+#include <numeric>
+#include <initializer_list>
+#include <cstdlib>
+#include <stdlib.h>
+#include <half.hpp>
+#include <math.h>
+#include "check_err.hpp"
+#include "config.hpp"
+#include "device.hpp"
+#include "host_tensor.hpp"
+#include "host_tensor_generator.hpp"
+#include "host_reduce_util.hpp"
+#include "host_reduction.hpp"
+
+#include "device_tensor.hpp"
+#include "device_gemm_xdl.hpp"
+#include "device_gemm_xdl_cshuffle.hpp"
+#include "element_wise_operation.hpp"
+#include "reference_gemm.hpp"
+#include "gemm_specialization.hpp"
+
+#include "device_reduce_blockwise.hpp"
+#include "reduction_enums.hpp"
+#include "reduction_operator_mapping.hpp"
+#include "device_binary_elementwise_2d.hpp"
+
+template <ck::index_t... Is>
+using S = ck::Sequence<Is...>;
+
+using F16 = ck::half_t;
+using F32 = float;
+
+using Row = ck::tensor_layout::gemm::RowMajor;
+using Col = ck::tensor_layout::gemm::ColumnMajor;
+
+using PassThrough = ck::tensor_operation::element_wise::PassThrough;
+
+using ADataType              = F16;
+using BDataType              = F16;
+using CDataType              = F16;
+using AccDataType            = F32;
+using EltwiseComputeDataType = F32;
+
+// CAUSION - host reduce_max will call numeric_limits<ck::half_t>::lowest()
```

Comment:
Typo: caution

Reply:
@rosenrodt If the comment is resolve, please click "Resolve conversation" button